pex/pep_427.py

```diff
@@ -394,13 +394,16 @@ def record_files(
         print("pex", file=fp)
     installed_files.append(InstalledWheel.create_installed_file(path=fp.name, dest_dir=dest))
 
-    if requested:
-        requested_path = os.path.join(dest, wheel.metadata_path("REQUESTED"))
-        touch(requested_path)
-        installed_files.append(
-            InstalledWheel.create_installed_file(path=requested_path, dest_dir=dest)
-        )
+    if interpreter:
```

Comment:
Why the change from unconditional to conditional for these files?

Reply:
REQUESTED is not required at all; see: https://peps.python.org/pep-0376/#one-dist-info-directory-per-installed-distribution.

RECORD is required to support uninstalling as noted; so we still include it when an interpreter is supplied (installing in a venv) for interoperability with external tools like `pip uninstall`, but not for installed wheel chroots since those are private and never uninstalled by anyone but Pex, and, since they are chroots, this can be done with a simple recursive chroot dir remove.

So, basically, this just saves a small amount of work for the most common Pex wheel install case.